hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -3046,6 +3046,8 @@ private void internalQueueCall(Call call, boolean blocking)
       } else {
         callQueue.add(call);
       }
+
+      LOG.debug("{} has entered the CallQueue and is waiting to be processed.", call);
```

Comment:
If only `callId` is output, there may be too little useful information. What do you think?

Reply:
Thanks @tomscut  for the comment and review.
The detailed log here is this:
`
2022-01-18 16:37:56,653 DEBUG org.apache.hadoop.ipc.Server: Call#75 Retry#0 org.apache.hadoop.hdfs.protocol.ClientProtocol.getFileInfo from xxxx.xxxx.xxxx.xxxx:port has entered the CallQueue and is waiting to be processed.
2022-01-18 16:37:56,654 DEBUG org.apache.hadoop.ipc.Server: Call#76 Retry#0 org.apache.hadoop.hdfs.protocol.ClientProtocol.getFileInfo from xxxx.xxxx.xxxx.xxxx:port has entered the CallQueue and is waiting to be processed.
`
When the Call is processed by Hanler, the detailed log is as follows:
`
2022-01-18 16:37:59,725 [5178363759] - WARN  [IPC Server handler 126 on 8025:Server@494] - Slow RPC : create took 509 milliseconds to process from client Call#76 Retry#0 org.apache.hadoop.hdfs.protocol.ClientProtocol.getFileInfo from xxxx.xxxx.xxxx.xxxx:port
`
In my opinion, the Calls recorded in these two places should be consistent.
@tomscut , do you have any good suggestions.

Reply:
I'm sorry I made a mistake. The output here is not only `callId`. And I think it is ok.

Reply:
The change to add a DEBUG log seems good, but how about this line instead?

  `LOG.debug("Call has entered the CallQueue and is waiting to be processed. Call details: {}", call);`

Usually we tend to read object's string presentation at the end to not mix up with the log statement, e.g RpcCall's toString() has word "from" and perhaps it becomes bit difficult to read the whole statement at the first sight, but if put at the end of the statement, we know that now we are only looking at RpcCall details:

```
    @Override
    public String toString() {
      return super.toString() + " " + rpcRequest + " from " + connection;
    }
```